apps/core/components/passengerForm/DateInput.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format, setMonth } from 'date-fns';
+import { View } from 'react-native';
+import {
+  StyleSheet,
+  Picker,
+  Text,
+  TextInput,
+} from '@kiwicom/universal-components';
+
+type Props = {|
+  +onBirthDateSubmit: Date => void,
+|};
+
+type State = {|
+  +day: string,
+  +month: string,
+  +year: string,
+  +dayError: string,
+  +yearError: string,
+|};
+
+type Month = {|
+  value: string,
+  label: string,
+|};
+
+class DateInput extends React.Component<Props, State> {
```

Comment:
I'm thinking about to have this component as a part of universal components. What do you think @KarinaDavtyan @FilipMessa ?

Reply:
Hmm I dont think its worth doing as multiplatform components development coming

Reply:
Hm, I think it is good Idea have it in the `universal-components` or inside `/margarita/packages/components/src/`

Reply:
True that it's somewhere between. But because it is "just" composite of multiple inputs, I'm slightly more for keeping it inside `margarita-components`.

Can we just maybe replace current `DateInput` inside `margarita-components` with this one? It existed only for this use case so the old one is no longer necessary.

Reply:
Alright, will replace DateInput with this